mapApp/views/restApi.py

```diff
@@ -1,33 +1,40 @@
-from mapApp.models import Incident, Hazard, Theft, Official, AlertArea, NewInfrastructure, Weather
-from mapApp import serializers as s
-from django.http import Http404
+import datetime
+
+from django.core.exceptions import ValidationError
+from django.contrib.auth import get_user_model
 from django.contrib.gis.geos import Polygon
+from django.http import Http404
+from django.views.decorators.csrf import csrf_exempt
+
 from rest_framework.views import APIView
 from rest_framework.response import Response
 from rest_framework import authentication, generics, permissions, status
-from django.views.decorators.csrf import csrf_exempt
+
+from mapApp import serializers as s
+from mapApp.models import Incident, Hazard, Theft, Official, AlertArea, NewInfrastructure, Weather
 from mapApp.permissions import IsOwnerOrReadOnly
-from push_notifications.models import GCMDevice, APNSDevice
 from mapApp.views import alertUsers, pushNotification
 
-from django.contrib.auth import get_user_model
-import datetime
+from push_notifications.models import GCMDevice, APNSDevice
+
 User = get_user_model()
 
 class CollisionList(APIView):
     """
     List all collisions, or create a new collision.
     """
     def get(self, request, format=None):
+        try:
+            # Extract bounding box Url parameter
+            bbstr = request.GET.get('bbox', '-180,-90,180,90')

```

Comment:
Consider pulling this out into a small helper function to reduce code duplication.